tox.ini

```diff
@@ -89,5 +89,4 @@ commands =
 [test-deps]
 deps =
     # If you need to test on a certain branch, add @<branch-name> after .git
-    git+https://github.com/aws/amazon-braket-schemas-python.git@quera
-    git+https://github.com/aws/amazon-braket-default-simulator-python.git
+    git+https://github.com/aws/amazon-braket-default-simulator-python.git@feature/neutral-atom-simulator
```

Comment:
This will cause the test to fail once we merge the `feature/neutral-atom-simulator` branch of amazon-braket-default-simulator-python to its `main` branch.

Please remove from the PR.

Reply:
Until we merge the default simulator feature branch, I recommend keeping this in (maybe add a todo to remove before merging to main), since otherwise the build will fail and then no more code changes can be verified on github until we launch

Reply:
Ok, I see. Makes sense. Thank you.

Reply:
@peterkomar-aws Ok, let me add a line
```# TODO: Remove the following line after merging the local sim to the main branch```